src/demo/app/fancy-form/fancy-field-demo.component.ts

```diff
@@ -0,0 +1,264 @@
+      </aside>
+
+      <aside>
+        <h2>no <code>model</code></h2>
+        <p>If model is undefined or not provided, the field will be disabled.</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field textinput name="nada" label="Nope"&gt;
+            &lt;div class="fancy-hint"&gt;This field is disabled because there is no model.&lt;/div&gt;
+          &lt;/prx-fancy-field&lt;
+        </pre>
+        Example:
+        <prx-fancy-field textinput name="nada" label="Nope">
+          <div class="fancy-hint">This field is disabled because there is no model.</div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>small</code></h2>
+        <p>Use the small parameter to get an <code>h4</code> label rather than <code>h3</code>.</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field small textinput [model]="model" name="foo" label="Foo"&gt;
+            &lt;div class="fancy-hint"&gt;Labels can be made small.&lt;/div&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field small textinput [model]="model" name="foo" label="Foo">
+          <div class="fancy-hint">Labels can be made small.</div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>inline</code></h2>
+        <p>Labels can be shown inline. Typically inline fields do not provide a hint.</p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field inline textinput [model]="model" name="foo" label="Foo"&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field inline textinput [model]="model" name="foo" label="Foo"></prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>required</code></h2>
+        <p>
+          In order for fields to actually be required, they must use the REQUIRED validator with their model.
+          The required parameter here adds the red * styling. That is all.
+        </p>
+        Usage:
+        <pre class="code">
+          &lt;prx-fancy-field required textinput [model]="model" name="mustProvide" label="First and Last Name"&gt;
+            &lt;div class="fancy-hint"&gt;Please provide your full name.&lt;/div&gt;
+          &lt;/prx-fancy-field&gt;
+        </pre>
+        Example:
+        <prx-fancy-field required textinput [model]="model" name="mustProvide" label="First and Last Name">
+          <div class="fancy-hint">Please provide your full name.</div>
+        </prx-fancy-field>
+      </aside>
+      
+      <aside>
+        <h2><code>.fancy-hint</code></h2>
+        <p>Use content projection with the <code>.fancy-hint</code> selector to show a field hint. Hints typically end with punctuation.</p>
+        Usage:
+        <pre class="code">
+          &lt;div class="fancy-hint"&gt;Hint text&lt;/div&gt;
+        </pre>
+        Example:
+        <prx-fancy-field textinput name="showhint" label="Hint">
```

Comment:
I think in the other examples, the Example matches the code shown in the Usage -- not here?